dts/arm/st/g4/stm32g4.dtsi

```diff
@@ -302,6 +302,54 @@
 			label = "SPI_3";
 		};
 
+		can {
```

Comment:
This leads to a warning during DTS generation:
```
board_name.dts.pre.tmp:281.7-324.5: Warning (simple_bus_reg): /soc/can: missing or empty reg/ranges property
```

Reply:
@mbolivar any suggestions on how we can improve the DT bindings?

Reply:
Maybe the top-level node could be set to the general CANFD configuration register?
```suggestion
		can: can@40006500 {
```

Reply:
It possibly miss a reg = <0>;